src/layouts/UpdatesPage.jsx

```diff
@@ -346,8 +343,39 @@ export const UpdatesPage = () => {
     });
   };
 
+  useEffect(() => {
+    const updateData = JSON.parse(localStorage.getItem('updateData'));
+    if (updateData) {
+      localStorage.setItem('updateData', null);
+      const keys = Object.keys(updateData);
+      keys.forEach((key) => {
+        if (key === 'representative') {
+          const selectedOption = selectRepresent.find((option) => {
+            if (option.value === updateData[key]) {
+              return option;
+            }
+          });
+          setRepresentativeSelected(selectedOption);
+        }
+        if (key === 'enteSelect') {
+          const selectedOption = selectInQuanto.find((option) => {
+            if (option.value === updateData[key]) {
+              return option;
+            }
+          });
+          setEnteSelected(selectedOption);
+        }
+        setTimeout(() => {
```

Comment:
I do not get the reason to use two `setTimeout()`

Reply:
because the page has been rendered without values in input fields, so with a little delay, the input fields has values visible